libs/plugins/base-href-rewrite/src/lib/plugins-base-href-rewrite.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  HandledRoute,
+  registerPlugin,
+  setMyConfig,
+  getMyConfig
+} from '@scullyio/scully';
+
+const baseHrefRewrite = async (
+  html: string,
+  route: HandledRoute
+): Promise<string> => {
+  const { href } = getMyConfig(baseHrefRewrite);
+  /** if there is a predicate and it returns falsy, don't do anything */
+  if (
+    route.config?.baseHrefPredicate &&
+    !route.config?.baseHrefPredicate(html, route)
+  ) {
+    return html;
+  }
+
+  console.log('heelo000');
```

Comment:
🙄

Reply:
the base-href is a WIP ;) I was aware of this.